ci-operator/jobs/openshift/installer/openshift-installer-presubmits.yaml

```diff
@@ -81,6 +80,59 @@ presubmits:
         - --template=/usr/local/e2e-aws
         - --target=e2e-aws
 
+  - name: pull-ci-origin-installer-e2e-aws-all
+    agent: kubernetes
+    context: ci/prow/e2e-aws-all
+    branches:
+    - master
+    rerun_command: "/test e2e-aws-all"
+    always_run: false
+    trigger: "((?m)^/test( e2e-aws-all),?(\\s+|$))"
+    decorate: true
+    skip_cloning: true
+    spec:
+      serviceAccountName: ci-operator
+      volumes:
+      - name: job-definition
+        configMap:
+          name: prow-job-cluster-launch-installer-e2e
+      - name: cluster-profile
+        projected:
+          sources:
+          - secret:
+              name: cluster-secrets-aws
+          - configMap:
+              name: cluster-profile-aws
+      containers:
+      - name: test
+        image: ci-operator:latest
+        volumeMounts:
+        - name: job-definition
+          mountPath: /usr/local/e2e-aws-all
+          subPath: cluster-launch-installer-e2e.yaml
```

Comment:
This is the same path used to define the `e2e-aws` job (which makes sense).  But depending on how long the full tests take to run, we might want to make the expiration date offset configurable.  #1103's 4 hours give a good buffer for `e2e-aws` (which takes ~40 minutes), but *this* job's full suite triggered [here][1] is at two hours and counting.

[1]: https://github.com/openshift/installer/pull/170#issuecomment-415832624